includes/Core/Admin/Screens.php

```diff
@@ -383,6 +383,25 @@ private function get_screens() {
 			)
 		);
 
+		$screens[] = new Screen(
+			self::PREFIX . 'user-input',
+			array(
+				'title'            => __( 'User Input', 'google-site-kit' ),
+				'capability'       => Permissions::MANAGE_OPTIONS,
```

Comment:
This also needs to pass the following so that the screen is not in the admin menu (per the ACs):
```suggestion
				'capability'       => Permissions::MANAGE_OPTIONS,
				'parent_slug'      => null,
```

Reply:
@felixarntz my mistake here, the IB says:
> args.parent_slug can be omitted as this will be null by default and will not appear in the admin menu.

I took omitted to mean "not added to the array" -  I have addressed this.

Reply:
@ryanwelcher Oh good point - I must have missed that when reviewing the IB.